src/containers/Home.jsx

```diff
@@ -24,8 +24,8 @@ const styles = StyleSheet.create({
     marginBottom: 15,
     color: theme.colors.white
   },
-  link: {
-    ...theme.text.link
+  link_dark_bg: {
```

Comment:
Maybe? I'm just guessing at the function of these styles, so I'm more generally suggesting the function drive the name.